src/ray/gcs/gcs_client_interface.h

```diff
@@ -0,0 +1,105 @@
+#ifndef RAY_GCS_GCS_CLIENT_H
+#define RAY_GCS_GCS_CLIENT_H
+
+#include <boost/asio.hpp>
+#include <memory>
+#include <string>
+#include <vector>
+#include "ray/common/status.h"
+#include "ray/gcs/actor_state_accessor.h"
+#include "ray/util/logging.h"
+
+namespace ray {
+
+namespace gcs {
+
+/// \class ClientOption
+/// GCS client's options(configuration items), such as service address, service password.
+class ClientOption {
```

Comment:
> maybe `GcsClientOptions` to be clearer?

gcs::ClientOption is fine to me

Reply:
1) should use plural `options`, because there're more than one option.
2) Although it's in `gcs` namespace, naming it `GcsClientOptions` would still be slightly better. because
  1) we may use an `Options` of another client, then it's confused that both classes are named `ClientOptions`. 
   2) it's more friendly to tools that ctags.